src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -123,6 +124,35 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchCurrentUser(currentUser);
     }
 
+    componentDidUpdate({ annotatorState: prevAnnotatorState }: Props): void {
+        const { action: prevAnnotatorAction } = prevAnnotatorState;
+        const {
+            annotatorState: { action: annotatorAction },
+        } = this.props;
+
+        if (annotatorAction !== prevAnnotatorAction) {
```

Comment:
Should we also check something unique to the annotation? Like maybe the unique id?